app/presenters/candidate_interface/personal_details_review_presenter.rb

```diff
@@ -82,6 +95,17 @@ def english_language_details_row
       }
     end
 
+    def right_to_work_row
+      if @nationalities_form.first_nationality != 'British' && @nationalities_form.first_nationality != 'Irish'
+        {
+          key: 'Residency status',
+          value: "#{formatted_right_to_work_or_study} <br> #{tag.p(@right_to_work_or_study_form.right_to_work_or_study_details)}".html_safe,
```

Comment:
Probably here should only show the details if they say they have the right to work 👌